src/test/unit/controls/Bubble/BubblePanning-spec.js

```diff
@@ -74,7 +73,7 @@ describe("Bubble - Panning", () => {
                 done();
             });
         });
-        it("Dynamic Data is updated correctly when key matches", () => {
+        it("Dynamic Data is updated correctly when key matches and label is retained when it is not passed", () => {
```

Comment:
It's a little confusing by reading the description.

It can be updated similarly in all test cases

```js
describe("when key matches", () => {
   describe("label is not passed", () => {
        it("should update dynamic data with label", () => {
           
        });
    });
});
```

Reply:
Updated [here](https://github.com/cerner/carbon-graphs/pull/290/commits/8bf2ea3397c96268c863b9c80909d4ae440ab9f7)